casanovo/config.yaml

```diff
@@ -117,7 +117,7 @@ save_model: True
 model_save_folder_path: ""
 # Set to "False" to save the PyTorch model instance
 save_weights_only: True
-# Model checkpointing frequency in training steps
+# Model validation and checkpointing frequency in training steps
```

Comment:
**question (non-blocking):** Do we want separate parameters for these? I haven't been training models so I don't know if this is sufficient or if they should be split.

Reply:
Separation would give flexibility to cases users who don't want to checkpoint as often as the validation but I think this keeps things simpler and works fine.